kinbot/kb.py

```diff
@@ -81,6 +81,15 @@ def main():
         logging.error('Specific reaction cannot be searched in PES mode.')
         return
 
+    uq = par.par['uq']
+    uq_n=1
+    if uq == 1:
+        uq_n = par.par['uq_n']
+        stPt_uq = par.par['stPt_uq']
```

Comment:
variable name needs to be changed if you changed in paramaters.py (just a note)

Reply:
resolved.